### ISSUE TITLE
[beta] Challenge won't pass test despite body font changing to Open Sans

### ISSUE THREAD

**Issue by Incredibl3dible** [freeCodeCamp/freeCodeCamp#12779]
Challenge [use-a-google-font](http://beta.freecodecamp.com/en/challenges/applied-visual-design/use-a-google-font) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.95 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```html
    
    
<style>
body {
  font-family: "Open Sans";
}
h4 {
  text-align: center;
  background-color: rgba(45, 45, 45, 0.1);
  padding: 10px;
  font-size: 27px;
  text-transform: uppercase;
}
p {
  text-align: justify;
}
.links {
  text-align: left;
  color: black;
  opacity: 0.7;
}
#thumbnail:hover {
  box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
}

.fullCard {
  width: 245px;
  border: 1px solid #ccc;
  border-radius: 5px;
  margin: 10px 5px;
  padding: 4px;
}
.cardContent {
  padding: 10px;
}
.cardText {
  margin-bottom: 30px;
}
</style>
<div class="fullCard" id="thumbnail">
<div class="cardContent">
  <div class="cardText">
    <strong><h4>Alphabet</h4></strong>
    <hr>
    <em><p>Google was founded by Larry Page and Sergey Brin while they were Ph.D. students at Stanford University.</p></em>
  </div>
  <div class="cardLinks">
    <u>
      <a href="https://en.wikipedia.org/wiki/Larry_Page" class="links">Larry Page</a><br><br>
      <a href="https://en.wikipedia.org/wiki/Sergey_Brin" class="links">Sergey Brin</a>
    </u>
  </div>
</div>
</div>

    ```

**Comment by dhcodes** [comment]
@Incredibl3dible thanks. Confirmed.

**Comment by HKuz** [comment]
Yes, the tests are checking for both the "Open Sans" part and the family-name (sans-serif). I can get the challenge to pass with both of those in the CSS rule. It's in the hint, but the test message should be clear about that, too. Maybe we revise the test to say something like: "The `body` tag should have a font of "Open Sans" and a family name of sans-serif, in that order".

**Comment by dhcodes** [comment]
@HKuz thanks. I think the issue is also with the Instructions that say:

>Set the font-family of the body to "Open Sans". 

without mentioning sans-serif so the instructions should probably be updated too.

Also, the CSS pattern may need to be updated to `'FAMILY-NAME', GENERIC NAME` per [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family)

Thoughts on this?

**Comment by HKuz** [comment]
@dhcodes agree on both counts, good call with matching the terminology to the MDN article. Based on your comment with their terms, how about the following changes (which come after the `link` example in the challenge text):

Once the link is in place, the font is available to include in your CSS by using the pattern `font-family: "FAMILY_NAME", GENERIC_NAME;`.

In this example, "FAMILY_NAME" would be "Open Sans" and GENERIC_NAME would be sans-serif. The quotes around "Open Sans" are required because it has a space in its name. Also, capitalization of the family name matters. Generic names are fallback fonts if the other specified font is not available. Also, generic names are keywords in CSS, so they should not have quotes.

Instructions
Set the `font-family` of the body to use a family name of "Open Sans" and generic name of sans-serif.

Test text: "The `font-family` property for the `body` tag should use a family name of "Open Sans" and a generic name of sans-serif, in that order".

Delete the hint

(Suggestions for any of that language are welcome)

**Comment by systimotic** [comment]
Just dropping by to say that this challenge seems like a bit of a duplicate of http://beta.freecodecamp.com/en/challenges/basic-css/import-a-google-font / https://www.freecodecamp.com/challenges/specify-how-fonts-should-degrade

**Comment by dhcodes** [comment]
I appeal to @HKuz

**Comment by HKuz** [comment]
Yeah, touche @systimotic  - I'm actually not sure what we're doing for very similar challenges. I know the JS section will have ones that cover certain topics across several sections. For example,  `map` and `slice` come up in BasicJS and Functional Programming, with slightly different spins on the material. I think for now we can keep them in - a little extra practice / redundancy / reinforcement of a concept can't hurt. If we start noticing inconsistencies, we can make a case for removing a specific challenge - does that sound somewhat reasonable?

**Comment by dhcodes** [comment]
Okay, I'll submit a PR for this tonight.
